visearch-android/src/main/java/com/visenze/visearch/android/model/Error.java

```diff
@@ -0,0 +1,31 @@
+package com.visenze.visearch.android.model;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Error {
+
+    @SerializedName("code")
+    private int status;
```

Comment:
why is this named "status", should be code

Reply:
changed already